test/test_helper.rb

```diff
@@ -1,6 +1,8 @@
 # Configure Rails Environment
 ENV["RAILS_ENV"] = "test"
 
+ENV["PDFJS_VIEWER_ORIGINS"] = "http://example.com,http://random.example.com"
```

Comment:
This should be set in the respective test-case and be reset after it (in an `ensure`)